src/wallet/rpc/wallet.cpp

```diff
@@ -198,7 +198,9 @@ static RPCHelpMan loadwallet()
     return RPCHelpMan{"loadwallet",
                 "\nLoads a wallet from a wallet file or directory."
                 "\nNote that all wallet command-line options used when starting bitcoind will be"
-                "\napplied to the new wallet.\n",
+                "\napplied to the new wallet."
+                "\nThe rescan is significantly faster if a descriptor wallet is loaded"
+                "\nand block filters are available (bitcoind option \"-blockfilterindex=1\").\n",
```

Comment:
Not sure it makes sense to talk about rescans on `loadwallet`. It can happen, but hopefully shouldn't be very often.

Reply:
Thanks, restored the original help text for `loadwallet` on the latest force-push.